website/docs/migration.md

```diff
@@ -0,0 +1,434 @@
+---
+id: migration
```

Comment:
Let's give it a more explicit name - `migration-from-v1-to-v2` or something. So that if we have a v2 to v3 migration in future there wouldn't be a naming collision.